hr_payslip_ytd_amount/README.rst

```diff
@@ -0,0 +1,72 @@
+Payslip Year-to-date Amount
```

Comment:
Hi, Thanks for your contribution.
I miss the license section here following https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst